hack/build-release.sh

```diff
@@ -3,6 +3,10 @@
 # This script generates release zips into _output/releases. It requires the openshift/origin-release
 # image to be built prior to executing this command via hack/build-base-images.sh.
 
+# NOTE, if you are making changes which operate in the builder pod as part of the builder image (for eaxmaple,
+# updates to processing of the build strategies), then you need to git commit those changes for them
+# to be picked up.  See hack/common.sh and look at the git related processing that occurs there.
+
```

Comment:
again i'd just make this a generic comment "only committed code is built by this script"

probably not necessary to add a comment to hack/build-images.sh since that's pretty clear what it does.